src/db/migrations/20190729144905-create-blacklist.js

```diff
@@ -0,0 +1,43 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('BooklistedToken', {
```

Comment:
Filename is `...create-blacklist` and table name is `BooklistedToken`? I'm confused.

Reply:
changed it to blacklistedTokens. Done.